zqd/handlers_zeek_test.go

```diff
@@ -51,7 +51,7 @@ func (s *PacketPostSuite) TestSpaceInfo() {
 	s.True(info.PacketSupport)
 }
 
-func (s *PacketPostSuite) SetupTest() {
+func (s *PacketPostSuite) SetupTestSuite() {
```

Comment:
This didn't cause an error? I believe `TearDownTest` runs after each test and deletes the root.

Reply:
actually it did! i got tripped up by the `// build zeek`tag. 
never mind, removing this commit and merging...

Reply:
I actually don't think this is bad idea to run this once for entire test suite (with an accompanying call to SuiteTearDown), being all tests don't change the spaces underlying state... it'll run faster. That said we can also add this back later.